src/User/Infra/Api/Security/ApiSecurityUserProvider.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\User\Infra\Api\Security;
```

Comment:
what about `Security\Jwt` instead? Im leaning to that because JWT seems more related to security then as any "api" no?

Nice work! Let me know when you want me to final review ;-)

Reply:
The bundle itself uses `Security` for a namespace.. i think it makes sense :)

Reply:
Ok why not, but what do you think about extends SecurityUserProvider from ApiSecurityUserProvider (or JWTSecurityUserProvider) ? I only need to write the loadUserByUsernameAndPayload function.

Reply:
`class MsgPhp\User\Infra\Security\Jwt\SecurityUserProvider implements UserProviderWithPayloadSupportsInterface`  that decorates `MsgPhp\User\Infra\Security\SecurityUserProvider` works for me :+1: